Pipfile

```diff
@@ -4,7 +4,7 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [packages]
-cognite-sdk = "==0.13.*"
+cognite-sdk = "==1.0"
```

Comment:
https://pypi.org/project/cognite-sdk/ latest version is at 1.4.10. Original ticket wording was not accurate, updated in azure devOps. please update to latest python sdk verison |> good news is that that sdks starting with 1.0 are backwards compatible so there shouldn't be a significant amount of changes.